dashboard/app/helpers/levels_helper.rb

```diff
@@ -132,9 +132,23 @@ def app_options
     view_options(is_channel_backed: true) if @level.channel_backed?
 
     return blockly_options if @level.is_a? Blockly
+    return dsl_defined_options if @level.is_a? DSLDefined
     Hash[view_options.map{|key, value|[key.to_s.camelize(:lower), value]}]
```

Comment:
Yep, might be clearer to change that to an plainer if instead of a series of fallthroughs

> On Aug 12, 2015, at 9:56, davidsbailey <notifications@github.com> wrote:
> 
> In dashboard/app/helpers/levels_helper.rb:
> 
> > @@ -132,9 +132,23 @@ def app_options
> >      view_options(is_channel_backed: true) if @level.channel_backed?
> >  
> >      return blockly_options if @level.is_a? Blockly
> > +    return dsl_defined_options if @level.is_a? DSLDefined
> >      Hash[view_options.map{|key, value|[key.to_s.camelize(:lower), value]}]
> Can you please add a comment saying which levels will hit this last case?
> 
> —
> Reply to this email directly or view it on GitHub.
>